src/main/java/io/citrine/jcc/search/pif/query/chemical/CompositionQuery.java

```diff
@@ -93,26 +99,6 @@ public CompositionQuery addOffset(final FieldQuery offset) {
     }
 
     /**
-     * Set the query to run against all fields.
-     *
-     * @param simple String with the query to run against all fields.
-     * @return This object.
-     */
-    public CompositionQuery setSimple(final String simple) {
-        this.simple = simple;
-        return this;
-    }
-
-    /**
-     * Get the query to run against all fields.
-     *
-     * @return String with the query to run against all fields.
-     */
-    public String getSimple() {
```

Comment:
Where did this go? Do we care about this?

Reply:
@sparadiso That was the first commit of the two. That method is defined in the parent class.